application.py

```diff
@@ -32,5 +35,41 @@ def contact():
     return render_template('contact.html')
 
 
+@application.route('/register', methods=['GET', 'POST'])
+def register():
+    form = RegistrationForm(request.form)
+    if request.method == 'POST' and form.validate():
+        # user = User(form.name.data,form.surename.data, form.email.data,
```

Comment:
what's that?

Reply:
place holder for the server side code. we can not enable it without implementing server side DB